maxim/logger/smolagents/client.py

```
@@ -0,0 +1,539 @@
+    Span,
+    SpanConfigDict,
+    ToolCall,
+    Trace,
+)
+from ...scribe import scribe
+
+# Global variables for tracking spans and state
+_last_llm_usages = {}
+_agent_span_ids = {}
+
+_global_maxim_trace: contextvars.ContextVar[Union[Trace, None]] = (
```

Ruff (0.12.2)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)